Source/drlg_l1.cpp

```diff
@@ -1710,32 +1710,32 @@ static void L5AddWall()
 	for (j = 0; j < DMAXY; j++) {
 		for (i = 0; i < DMAXX; i++) {
 			if (!L5dflags[i][j]) {
-				if (dungeon[i][j] == 3 && random_(0, 100) < 100) {
+				if (dungeon[i][j] == 3 && GenerateRnd(100) < 100) {

```

Comment:
The GenerateRnd(100) < 100 seems pretty pointless, as GenerateRnd(100 will only generate numbers from 0 - 99, unless I'm mistaken? There's a lot of those calls.

Reply:
You are correct...

Reply:
I'm against removing first argument as without it you basically can't debug rng calls, it's used to track which part of the code called it